util/httputil/compression.go

```diff
@@ -30,51 +31,31 @@ const (
 
 // Wrapper around http.Handler which adds suitable response compression based
```

Comment:
It's not around `http.Handler`, and doesn't look at headers.

Reply:
good catch! thanks a lot for the comments btw!